dev-libs/elftoolchain/elftoolchain-0.7.1.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Libraries/utilities to handle ELF objects (BSD drop in replacement for libelf)"
+HOMEPAGE="https://wiki.freebsd.org/LibElf"
+SRC_URI="https://netcologne.dl.sourceforge.net/project/elftoolchain/Sources/${P}/${P}.tar.bz2"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	app-arch/libarchive:*
+	dev-libs/expat
+	dev-libs/libbsd
+	!dev-libs/elfutils
+	!dev-libs/libelf"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-vcs/subversion
+	sys-apps/lsb-release
+	sys-devel/bmake
+	virtual/yacc"
+
+# Requires network access and otherwise fails
+RESTRICT="test"
```

Comment:
Any chance of patching those out or pre-downloading the files, or too much work?

Reply:
I just patched the tests out from main Makefile now, we still need `RESTRICT` clause, is that correct?

Reply:
That completely depends on what happens when it tries to run the test phase. If it skips because there's no target, then no need for RESTRICT (although in some sense it's nice as a note to actually fix tests, I'd just add "# NOTE: Need to fix tests ...")